service/src/main/java/bio/terra/tanagra/app/controller/StudiesApiController.java

```diff
@@ -84,14 +84,17 @@ public ResponseEntity<ApiStudyList> listStudies(
       String createdBy,
       Boolean includeDeleted,
       List<String> properties,
+      Boolean allStudies,
       Integer offset,
       Integer limit) {
     ResourceCollection authorizedStudyIds =
-        accessControlService.listAuthorizedResources(
-            SpringAuthentication.getCurrentUser(),
-            Permissions.forActions(STUDY, READ),
-            offset,
-            limit);
+        allStudies
+            ? ResourceCollection.allResourcesAllPermissions(STUDY, null)
```

Comment:
A bit concerned bypassing the access control altogether - an unauthorized caller can get studies they should not have access to by passing this flag. This would affect all access control models in all deployments. 

- is there a particular admin user that would call this api to get all studies? 
- how about a new function in accessControlService.listAllResources(adminUser, permissions, offset, limit). This could check that the user requesting the list is an admin user and would return ALL resources or all users (subject to offset / limit)? The default behavior in the interface could be return none, and access control model impl can override it.

Reply:
Another very minor issue is that we also ignore the offset / limit. can the caller handler the large response of all studies, assuming that backend service will not have issues with this large data? verily unlikely that we run out of memory for this API, but putting it out there.

Also, could there be a case where the sd/emerge admin caller wants the response paginated? 
instead, is it feasible to send a large limit (int max) in the API request? Another alternate is to set limit=int.Max if it is not set in the request, and its is an admin caller (see comment on access control for admin caller)

This way we retain the support for offset / limit.

Reply:
After discussing with @chenchals, we agree that handling this in the admin api is the better option. Will either modify `listAuthorizedResources` to check if user is admin or add new endpoint for that check. Leaving this pr open for now in case core Tanagra changes will be needed .